### ISSUE TITLE
Electron build(s) broken in various ways

### ISSUE THREAD

**Issue by joshuaauerbachwatson** [IBM/kui#612]
I have not yet found a (packaged) electron build sequence that fully works.

I did
```
git clone https://github.com/IBM/kui.git
cd kui
npm install
cd clients/default
npm run build:electron
cd dist/electron/Kui-darwin-x64
open Kui,app
```
The result has no left tab stripe and other outermost areas seem to lack their usual coloration.

I then built a rudimentary external client as per the instructions and ran its electron build:
```
mkdir my-client && cd my-client
npm init -y
npm install --save-dev @kui-shell/builder
npm install --save @kui-shell/core
npm install --save @kui-shell/plugin-k8s
npx kui-init
npx kui-build-electron
cd dist/electron/Kui-darwin-x64
open Kui,app
```
This results in a pop-up error panel:
<img width="309" alt="screen shot 2019-03-02 at 2 59 53 pm" src="https://user-images.githubusercontent.com/19649275/53686942-e6e90700-3cfb-11e9-87d2-281d94c1fe55.png">

 My real goal here is to get the electron build to work in my own custom external client.    When I run the electron build there (similar to the above) and open the resulting app, a window shows but doesn't fully initialize.   The error in the console is
```
Uncaught Error: Cannot find module 'debug'
    at Module._resolveFilename (internal/modules/cjs/loader.js:584)
    at Function.Module._resolveFilename (/Users/joshuaauerbach/nimbella/workbench/nimbellaClient/dist/electron/Serverless Workbench-darwin-x64/Serverless Workbench.app/Contents/Resources/electron.asar/common/reset-search-paths.js:43)
    at Function.Module._load (internal/modules/cjs/loader.js:510)
    at Module.require (internal/modules/cjs/loader.js:640)
    at require (internal/modules/cjs/helpers.js:20)
    at electron.ts:17
nimbella.svg:1 Failed to load resource: net::ERR_FILE_NOT_FOUND
```
The `nimbella.svg` referred to is an icon that we use.

**Comment by starpit** [comment]
thanks for the bug reports!

the first issue is covered by #406 

the second is a separate issue. the problem here is that the core has a non-catching dependence on plugin-core-support. we should catch this error, as it is innocuous in the absence of theme support (which comes with plugin-core-support)

**Comment by starpit** [comment]
the debug and theme issues (second two issues) should be resolved by #616 

i'll close this, as the remaining (first) issue is covered by #406
